python/lsst/meas/algorithms/stamps.py

```diff
@@ -33,9 +33,11 @@
 import lsst.afw.fits as afwFits
 from lsst.geom import Box2I, Point2I, Extent2I, Angle, degrees, SpherePoint
 from lsst.daf.base import PropertyList
+from lsst.daf.butler.core.utils import getFullTypeName
+from lsst.utils import doImport
 
 
-def writeFits(filename, stamp_ims, metadata, write_mask, write_variance):
+def writeFits(filename, stamp_ims, metadata, type_name, write_mask, write_variance):
```

Comment:
@SimonKrughoff should we start writing a version number to these files?

Reply:
I guess so since we are breaking the API, but does that mean we have to also check the version and fall back to the old API when `STAMPCLS` is missing?  Seems like a lot of overhead, but if Morgan needs to read old stamps with new code, it will cause a problem.  I don't have any old STAMPS around, so am happy to just move on with the new order

Reply:
For now adding the VERSION won't have much effect because the only thing that really matters is STAMPCLS being present or not. If STAMPCLS is missing we can only complain and can't do anything to change that with versioning.  I'm more thinking about a future world where we decide to make another change and it would be handy if we could have a VERSION=1 vs VERSION=2 header (absence meaning VERSION == 0).  We've shown in our serialization of Exposure that versioning does become useful eventually.

Reply:
Fair enough, then I'm happy to take your suggestions

Reply:
Ok, I updated to add a version number in the metadata. For now is that all I should do?

Reply:
Yes, seems fine.